doc/running.rst

```diff
@@ -163,3 +186,11 @@ To generate code that is compatible with particular release versions of NEST Sim
 - The default is the empty string, which causes the NEST version to be automatically identified from the ``nest`` Python module.
 - ``"v2.20.2"``: Latest NEST 2 release.
 - ``"master"``: Latest NEST GitHub master branch version (https://github.com/nest/nest-simulator/).
+
+
+Python-standalone target
+------------------------
+
+*NESTML features supported: neurons*
```

Comment:
Could you also mention the features supported currently for the python-standalone target? It would also be beneficial to mention the list of files generated for the target and how to run the simulation after.